egs/wenetspeech/ASR/pruned_transducer_stateless2/decode.py

```diff
@@ -274,6 +274,7 @@ def decode_one_batch(
         hyp_tokens = greedy_search_batch(
             model=model,
             encoder_out=encoder_out,
+            encoder_out_lens=encoder_out_lens,
```

Comment:
Can you also fix fast_beam_search? I think its name has been changed to fast_beam_serach_one_best.

Also, for the following `modified_beam_search`, it also needs `encoder_out_lens`.